workflows/snvphylnfc.nf

```diff
@@ -169,10 +169,19 @@ workflow SNVPHYL {
     CAT_CAT(invalid_positions)
 
     invalid_positions_file = CAT_CAT.out.file_out.map{meta, filepath -> filepath}
+    consolidated_bcfs = CONSOLIDATE_BCFS.out.consolidated_bcfs.toSortedList{a, b -> a[0].id <=> b[0].id}
 
-    //13. consolidate variant calling files process takes 2 input channels as arguments
+    // consolidated_bcfs is a list of [meta, filepath] tuples
+    // .map{} iterates onces, because there is only one item (one list) in the channel
+    // .collect{} transforms each item in the list with the operation
+    // which ultimately unzips the tuples
+    consolidated_bcfs_metas = consolidated_bcfs.map{ it.collect { it[0] } }
```

Comment:
Do you need `map { it.collect { it[0] } }` or would this also work with `consolidated_bcfs.collect { it[0] }`?

Reply:
As discussed, but briefly:

The first `.map{}` iterates over the 1 item in the `consolidated_bcfs` channel, which is a list of `tuple(val(meta), path(filepath))`. The following `.collect{}` iterates over the items in that list.

However, I've changed the first `.map{}` to a `.collect{}`, since it is closer to the behaviour we want (value channel instead of queue channel).

Reply:
Awesome. Thanks so much.